In [1]:
import pandas as pd

datos = {
    "pais": ["España", "Francia", "Italia", "Alemania"],
    "PIB": [1400, 2800, 2100, 3800],
    "inflacion": [2.5, 2.1, 2.8, 1.9]
}

df = pd.DataFrame(datos)

print(df["PIB"].corr(df["inflacion"]))

-0.8093086866544134


## Existe una correlación negativa fuerte (-0.81) entre el PIB y la inflación. Esto indica que, en estos datos, cuando el PIB aumenta, la inflación tiende a disminuir. Sin embargo, esta conclusión debe interpretarse con cautela, ya que el análisis se basa únicamente en 4 países, lo que puede hacer que la correlación sea poco representativa.